charts/minecraft/templates/minecraft-svc.yaml

```diff
@@ -40,6 +40,8 @@ spec:
   - name: {{ .name }}
     port: {{ .service.port }}
     targetPort: {{ .name }}
+    {{- if .service.nodePort }}
```

Comment:
This one is also missing an `end`

Reply:
Thanks!  Don't know how I missed that one.